contracts/MetadataViews.cdc

```diff
@@ -415,4 +415,55 @@ pub contract MetadataViews {
             self.socials = socials
         }
     }
+
+    // An NFT can add MetadataViews.Rarity as a top-level view to define the total rarity.
+    pub struct Rarity {
+        pub let score: UFix64?
+        pub let description: String?
+
+        init(score: UFix64?, description: String?) {
+            self.score = score
```

Comment:
Should we assert here that you need to atleast specify one?

Reply:
Probably we should sync the rarity struct you defined in #92 here?

Reply:
Sync'd this PR with #92